photon-core/src/main/java/org/photonvision/common/dataflow/websocket/UILightingConfig.java

```diff
@@ -0,0 +1,11 @@
+package org.photonvision.common.dataflow.websocket;
+
+public class UILightingConfig {
```

Comment:
Should this be UILightingConfiguration to match the naming convention of the other ones? Or, vice-versa should PhotonConfiguration, UICameraConfiguration, and UiPhotonConfiguration be shortened to ...Config?

Reply:
I'm really not sure. I am open to suggestions on structure, not super happy with the current thing.

Reply:
Is that suggestive of a larger settings refactor/reorganization?

Reply:
There's other places where we use raw hashmaps instead of using classes; this is just the biggest offender. I'm happy to split things up into more files, or keep everything in one place so it reads more like the frontend timescript types do.

Reply:
Yeah idk. I can see arguments either way tbh. Do you have opinions?

Reply:
I think this is a good start and we can pick away at the others over time.